.env-dist

```diff
@@ -1,5 +1,6 @@
 # local, heroku, stage, production
 APP_ENV=local
+NEXT_PUBLIC_APP_ENV=local
```

Comment:
We’ve reverted most of our usage of `NEXT_PUBLIC_*` env variable. In this case, I think it might be nice to have this one it’s very straightforward to add `useTelemetry` to any client component without having to pass around the props needed to initialize Glean.

Reply:
We didn't remove them because we didn't like them, but because they get compiled in at build time, and thus can't differ per environment. So they're fine to use when we're hardcoding them into `.env-dist`, but for a use case like this, it unfortunately won't work.

Reply:
Yes unfortunately the build happens in CI, so any `NEXT_*` values are compiled into the JS and then published inside a docker container which gets reused for both stage/prod.

Reply:
I’ve [190c101](https://github.com/mozilla/blurts-server/pull/3813/commits/190c101f6991140e58894ff95dc227cf7b46ce24) a `<PublicEnvProvider>` to help us out here.